pkg/security/clihelpers/eval.go

```diff
@@ -186,6 +174,68 @@ func eventDataFromJSON(file string) (eval.Event, error) {
 	return event, nil
 }
 
+func variablesFromTestData(testData TestData) (map[string]eval.SECLVariable, error) {
+	variables := make(map[string]eval.SECLVariable)
+
+	// copy the embedded variables
+	for k, v := range model.SECLVariables {
+		variables[k] = v
+	}
+
+	for k, v := range testData.Variables {
+		switch v := v.(type) {
+		case string:
```

Comment:
Should we also support []string and []int (and use NewScopedStringArrayVariable and NewScopedIntArrayVariable) ?

Reply:
@lebauce indeed. I just pushed a commit to handle them.